src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.macios.cs

```diff
@@ -92,14 +97,32 @@ public partial class MediaManager : IDisposable
 		{
 			Player = Player
 		};
-
 		// Pre-initialize Volume and Muted properties to the player object
 		Player.Muted = MediaElement.ShouldMute;
 		var volumeDiff = Math.Abs(Player.Volume - MediaElement.Volume);
 		if (volumeDiff > 0.01)
 		{
 			Player.Volume = (float)MediaElement.Volume;
 		}
+		if (Dispatcher.IsDispatchRequired)
```

Comment:
`Dispatcher.Dispatch` implementation doesn't do this check already
?

Reply:
I am following the existing pattern previously used in Media Element in `MediaManager.windows.cs` on line 98 to 110. The method name is `protected virtual async partial Task PlatformSeek(TimeSpan position, CancellationToken token)` If that is incorrect or my implementation is wrong I am happy to fix it.